### ISSUE TITLE
Abnormal circulating Cathepsin D concentration

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#5992]
**Preferred term label:** Abnormal circulating Cathepsin D concentration

**Synonyms**

**Definition (free text, please give PubMed ID)** Concentration of cathepsin D in the blood is outside the normal range., PMID: 18296260

**Parent term (use hpo.jax.org/app)** Abnormality of circulating enzyme level HP:0011021

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by JillMiller** [comment]
**Preferred term label:** Decreased circulating Cathepsin D concentration

**Synonyms**

**Definition (free text, please give PubMed ID)** Concentration of cathepsin D in the blood is below the normal range.

**Parent term (use hpo.jax.org/app)** Abnormal circulating Cathepsin D concentration

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by JillMiller** [comment]
**Preferred term label:** Increased circulating Cathepsin D concentration

**Synonyms**

**Definition (free text, please give PubMed ID)** Concentration of cathepsin D in the blood is above the normal range.

**Parent term (use hpo.jax.org/app)** Abnormal circulating Cathepsin D concentration

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Only the "increased" term is medically relevant. Added as HP_0033182
